Controller/AuthorizeController.php

```diff
@@ -52,14 +52,14 @@ public function authorizeAction(Request $request)
         $form = $this->container->get('fos_oauth_server.authorize.form');
         $formHandler = $this->container->get('fos_oauth_server.authorize.form.handler');
 
+        $scope = $request->query->get('scope', $request->request->get($form->getName().'[scope]', null, true));
```

Comment:
The way you get scope from the form should not be hardcoded: `$form->getName().'[scope]'`